lib/provider/immoscout-mobile.js

```diff
@@ -0,0 +1,161 @@
+import utils, {buildHash} from '../utils.js';
```

Comment:
Can you please add some documentation here?

Also the old immoscout implementation does not work anymore, thus, if this one works, it can replace the current implementation.